iceberg/src/main/java/com/nvidia/spark/rapids/iceberg/spark/source/StructInternalRow.java

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.iceberg.spark.source;
```

Comment:
This class is shimmed in iceberg https://github.com/apache/iceberg/blob/main/spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/StructInternalRow.java

We should add  shimplify annotations to list the Spark versions it supports.
```
/*** spark-rapids-shim-json-lines
*/
```

Reply:
Sounds good as long as you have an issue to track it

Reply:
We need to remove the module from all but release35X profiles